libraries/core-react/src/Slider/Slider.tsx

```diff
@@ -183,9 +190,9 @@ export type Props = {
   minMaxValues?: boolean
   /** Disabled */
   disabled?: boolean
-} & JSX.IntrinsicElements['div']
+} & Omit<HTMLAttributes<HTMLDivElement>, 'onChange'>
```

Comment:
Got a mismatch between our `onChange` type and `HTMLDivElement` `onChange` type... So had to remove it from `HTMLDivElement` type import to clear TS errors

Reply:
Can we change the sliders `onChange` to match the native behaviour?

Reply:
@mimarz I've been playing a lot with this, check out this weird error when I update our `onChange` type to match native type. No matter how our `onChange` type looks like, we get the error `'(event: blablabla is not assignable to type '((event: ...` It looks like they get extra brackets somewhere!  
![Screenshot 2020-11-03 at 15 02 19](https://user-images.githubusercontent.com/31915755/97994489-a5b8cc80-1de5-11eb-80ce-b6e3a4e24d37.png)

Reply:
No wonder it does not work with native `onChange` because it expects only one parameter, not two, which we have override/changed 🙈

![image](https://user-images.githubusercontent.com/1070981/98081179-f0365980-1e76-11eb-9c99-f090dc32243d.png)

Reply:
@mimarz 🤯  I don't have that error in my branch

Reply:
Btw all errors cleared now after latest commit, merge or do you wan't to review first @mimarz ?

Reply:
I'll have a look @pomfrida  :)